goFundMo-app/src/App.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './App.css';
+import TopBar from './Components/TopBar';
+import Progress from './Components/Progress';
+import DonationForm from './Components/DonationForm';
+
+let donations = [];
```

Comment:
If you wanted to have state on this file, you could have made it a Component instead of a function